source/kraken/realtime.cpp

```diff
@@ -640,8 +641,11 @@ void handle_realtime(const std::string& id,
                                 << get_wordings(get_trip_effect(trip_update.GetExtension(kirin::effect))));
         return;
     }
-
+    auto begin = pt::microsec_clock::universal_time();
     const auto* disruption = create_disruption(id, timestamp, trip_update, data);
+    LOG4CPLUS_DEBUG(log4cplus::Logger::getInstance("logger"),
+                    "it took " << (pt::microsec_clock::universal_time() - begin).total_milliseconds()
+                               << " ms to create disruption " << id);
```

Comment:
Passive voice + adding single quotes around `id`.

```suggestion
                    "creating disruption '" << id << "' took " << (pt::microsec_clock::universal_time() - begin).total_milliseconds()
                               << " ms");
```